python_modules/dagster/dagster/utils/log.py

```diff
@@ -239,3 +218,41 @@ def quieten(quiet=True, level=logging.WARNING):
     finally:
         if quiet:
             logging.disable(logging.NOTSET)
+
+
+def configure_loggers(allow_handler_overrides=False, log_level="INFO"):
```

Comment:
should `allow_handler_overrides=False` be `dagster_handler=default`?

Do we still need the 'null' handler in the 'default' case and vice versa? If not, maybe we let people pass in handler config to override instead?

Reply:
refactored to be the literal type